Add Gradio interface for inpainting

leszekhanusz opened the pull request:
It looks like this:

![gradio_inpainting_screenshot](https://user-images.githubusercontent.com/4641499/185757990-5e58f0d4-cd4d-456b-844c-ddbf173f9b42.png)

leszekhanusz committed:
Add Gradio interface for inpainting

osanseviero commented:
This is very cool! Is there a Hugging Face Space for this? We would be happy to provide temporary T4 GPUs for it to run faster :)

leszekhanusz committed:
Improvements for images not 512x512 pixels

- Now creating an image the same size of input by resizing prediction
- Automatically rotate the image according to EXIF data

leszekhanusz commented:
No, it is not on Hugging Face yet.

leszekhanusz commented:
The gradio interface was a little bit limitating so I made a web frontend using Vue to test the inpainting of Stable Diffusion with diffusers.
It is available [here](https://github.com/leszekhanusz/diffusion-ui) and [looks like this for now](https://imgur.com/a/KMR8tJf)
It is still a work in progress but you can already generate images and draw inpainting mask to generate editions. Please tell me what you think.

osanseviero commented:
This is very very cool! It looks nice :D thanks for sharing 🔥
